pkg/skuba/actions/node/join/join.go

```diff
@@ -98,6 +99,14 @@ func Join(client clientset.Interface, joinConfiguration deployments.JoinConfigur
 		if err := cni.CiliumUpdateConfigMap(client); err != nil {
 			return err
 		}
+
+		newReplica, err := replica.NewHelper(client)
+		if err != nil {
+			return err
+		}
+		if err := newReplica.UpdateNewControlPlaneNodes(); err != nil {
```

Comment:
I would suggest to do the replica and affinity update when node change not limited to control plane node except any reason?

Reply:
I was too tide up on the master from original issue description. Updated, thanks for the catch.